crates/transaction-pool/src/error.rs

```diff
@@ -169,6 +173,8 @@ impl InvalidPoolTransactionError {
                     InvalidTransactionError::GasUintOverflow => true,
                     InvalidTransactionError::TxTypeNotSupported => true,
                     InvalidTransactionError::SignerAccountHasBytecode => true,
+                    #[cfg(feature = "optimism")]
+                    InvalidTransactionError::PooledDepositTx => true,
```

Comment:
should this be "InvalidPoolTransactionError" instead of "InvalidTransactionError?  (like on L188)